qrcode/project.yml

```diff
@@ -4,3 +4,5 @@ packages:
       - name: qr-java 
         runtime: java:default
         main: qr.Generate#main
+      - name: qr 
+        runtime: tessjs:10
```

Comment:
```suggestion
        runtime: nodejs:default

```

Reply:
the `qr` -> `nodejs` mapping is not necessary unless you want to change the default runtime mapping. I don't think the `tessjs` runtime which is used for machine learning/tesseract app is necessary here.

So the `qr` stanza isn't needed but it's ok to add now that we have a `project.yml` file anyway.